analysis/demonhunterhavoc/src/CONFIG.js

```diff
@@ -1,14 +1,15 @@
+import { ToppleTheNun } from 'CONTRIBUTORS';
 import Expansion from 'game/Expansion';
 import SPECS from 'game/SPECS';
 
 import CHANGELOG from './CHANGELOG';
 
 export default {
   // The people that have contributed to this spec recently. People don't have to sign up to be long-time maintainers to be included in this list. If someone built a large part of the spec or contributed something recently to that spec, they can be added to the contributors list. If someone goes MIA, they may be removed after major changes or during a new expansion.
-  contributors: [],
+  contributors: [ToppleTheNun],
```

Comment:
Just so you know, you aren't required to put yourself here. If you do, you'll probably get people asking for more changes. If you're fine with that and intend to add more Havoc DH stuff then certainly you can leave this here.

Reply:
I intend to do some more maintenance of the spec; there's some things that are missing that I'd love to improve.